assets/section-multicolumn.css

```
@@ -245,3 +249,14 @@
     padding-right: 1.5rem;
   }
 }
+
+.multicolumn-card__info .link {
+  text-decoration: none;
+  font-size: inherit;
+  margin-top: 1.5rem;
+}
+
+.multicolumn-card__info .link .icon-wrap {
```

```suggestion
.multicolumn-card__info .icon-wrap {
```

You might be able to get rid of the `.link` specificity here, but I'll leave that up to you to decide.